src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,56 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      this.die();
+    }
```

Comment:
The `checkHealth` method is not necessary as the health check and removal from the `Animal.alive` array can be done directly within the `bite` method of the `Carnivore` class. This would simplify the code and avoid the need for an extra method call.